openspec/changes/add-drift-breaker-semantics/tasks.md

```
@@ -0,0 +1,33 @@
+## 1. Drift self-heal attempt bounding (#259)
```

markdownlint-cli2 (0.22.1)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)